sys/cddl/contrib/opensolaris/uts/common/dtrace/dtrace.c

```diff
@@ -3224,7 +3232,7 @@ dtrace_speculation_buffer(dtrace_state_t *state, processorid_t cpuid,
  * dtrace_dif_variable() uses this routine as a helper for various
  * builtin values such as 'execname' and 'probefunc.'
  */
-uintptr_t
+uintcap_t
```

Comment:
Note that the local variable `ret` here is still an `uintptr_t` so the return value will not be a valid capability.